README.md

```diff
@@ -3,18 +3,18 @@
 
 This package models YouTube Analytics data from [Fivetran's connector](https://fivetran.com/docs/applications/youtube-analytics#youtubeanalytics). It uses data in the format described by [this ERD](https://fivetran.com/docs/applications/youtube-analytics#schemainformation).
```

Comment:
YouTube Analytics doesn’t have an ERD. Can we rephrase this?

Reply:
I think we phrased it best in the source pacakge 
```txt
It uses data in the format described by the [YouTube Channel Report schemas](https://fivetran.com/docs/applications/youtube-analytics#schemainformation).
```
I will update this section to reflect the source pacakge.

Reply:
Agreed - thanks for making the change!